### ISSUE TITLE
Upgrade to Express 4

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#164]
@mattpen noted in https://github.com/phetsims/perennial/issues/47#event-1344059690 that Express 4 is available and has been for a while.  For future maintainability, we should upgrade the version that Rosetta is using, which is 3.4.8.

**Comment by jbphet** [comment]
It would probably be wise to do this very early in the chipper:2.0 work, so I'm making it top priority.

**Comment by jbphet** [comment]
On https://expressjs.com/en/guide/migrating-4.html, they say:

> Express 4 is a breaking change from Express 3. That means an existing Express 3 app will not work if you update the Express version in its dependencies.

**Comment by jbphet** [comment]
I followed the guide that can be found in the link in the comment just above, and it was a fairly quick process, requiring only a few minor changes to the code.  I tested on ox-dev, and verified that I could log in, make changes to a translation and test it (using the "Test" button), save strings in the temporary DB, and submit string and have them show up in babel.  Since that all worked, I went ahead and did the same process on phet-sever (after first verifying that no one was in the process of doing a translation).  So, this is now live on the main site, and it all seems to be working fine.  I'll monitor the log and make sure that we get at least one successful translation before I close this.
